renovate.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended"
+  ],
+  "ignorePaths": [],
+  "labels": ["Skip Changelog"],
+  "separateMajorMinor": true,
```

Comment:
```suggestion
```

nit. This was enabled by default. https://docs.renovatebot.com/configuration-options/#separatemajorminor